README.md

```diff
@@ -1,43 +1,141 @@
+
+To install the modules, run this command with <my_project_dir> of your own choosing:
+
+```sh
+cdf-tk init <my_project_dir> 
+```
+
+
+### Select which modules to deploy
+
+Edit the `<my_project_dir>/local.yaml` file. You can add multiple environments (Cognite Data Fusion Projects[^1]) like this:
+
+```yaml
+# environment name
+dev:
+  project: <customer>-dev # the cdf project name
+  type: dev
+  deploy: # packages or modules to deploy to this environment
+    - cdf_demo_infield 
+    - cdf_apm_simple
+```
+
+The environment (dev in the example) is used by the cli tool as `--env <environment>` to determine which configurations to deploy.
 
-Below is an overview of the scope of what can be governed through using these templates:
 
-![Overview of project templates](./static/overview.png "Overview")
 
-## Quickstart
+### Configure each module as needed
 
-To install the `cdf-tk` tool, you need a working Python installation >=3.9 (recommended 3.11).
-Run: `pip install cognite-toolkit`
+Each module follow a similar pattern:
 
-The `cdf-tk` tool is available as a command line tool. Run `cdf-tk --help` to see the available commands.
+```
+./modules|common|examples|local_modules/<moduleA>/
+                                                |- data_models/
+                                                |- auth/
+                                                |- transformations/
+                                                |- raw/
+                                                |- time_series/
+                                                |- files/
+                                                default.config.yaml 
+                                                config.yaml
+```
 
-## For more information
+Create a copy of default.config.yaml to config.yaml and edit it according to the customer setup. In general, this is the only file that should be edited in the modules. The build command will replace placeholder values in the templates with these values.     
 
-More details about the tool can be found at
-[developer.cognite.com](http://developer.cognite.com/sdks/toolkit).
+> [!WARNING]
+>
+> `default.config.yaml` files and the contents in the resource folders should not be edited as they might be overwritten by toolkit updates.
+> The correct way to configure a module is to make a copy of `default.config.yaml` as `config.yaml` and make changes there. These files should be added to version control.
+
+
+#### 2.3 Create your own custom modules
+
+Following the same pattern as above, custom modules can be added to `<my_project_dir>/local_modules/`. If you add these modules to your local.yaml, they will get included in the build. See advanced usage of templates in the [Developer Toolkit Advanced usage of templates](https://developer.cognite.com/sdks/toolkit/advanced).
+
+
+
+## Step 3: Build, verify and deploy the configuration
+
+Run the `build` command with the `cdf-tk` CLI:
+
+```sh
+cdf-tk build --env <my_env> --build-dir <my_build_dir> 
```

Comment:
You should include the source_dir as an argument maybe? Or comment that you need to run cdf-tk from the project dir. Maybe drop --build-dir as it will default be ./build?